jsonnet/jsonnet.bzl

```diff
@@ -115,12 +160,12 @@ def _jsonnet_to_json_impl(ctx):
        "-J %s" % ctx.genfiles_dir.path,
        "-J %s" % ctx.bin_dir.path] +
       ["--ext-str %s=%s"
-       % (_quote(key), _quote(ctx.var[val])) for key, val in jsonnet_ext_strs.items()] +
+       % (key, _quote(val)) for key, val in jsonnet_ext_strs.items()] +
```

Comment:
Why is it safe to not quote the key?

Reply:
Good question - I will take a look at that again.

Reply:
You are right, we should quote keys, in case they contain quotes. I added a test case for this.